app/models/season/phase_switcher.rb

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
   def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    if phase.in?(whitelisted_phases)
+      self.public_send(phase)
+    end
+  end
+
+
+  private
+
   def self.fake_coding_phase
     season.update({
```

Comment:
Since we're on Ruby > 2.some, I'd vote for using *"real"* KWARGS everywhere. Plus, since it's only one param, this can also savely be done in one line I'd say:
```ruby
season.update(starts_at: 6.weeks.ago)
```